testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -264,6 +264,25 @@ def test_deprecated_warning(self, hole, attrname):
             value = getattr(hole, attrname)
             assert value is hole.results[attrname]
 
+
+class TestOserror(BaseTestHole):
```

Comment:
I wouldn't subclass `BaseTestHole` here, it adds a set of tests via `test_deprecated_warning` which aren't necessary to re-test here. Instead just add a fixture that does something like this:

```python
@pytest.fixture()
def universe(self):
    return mda.Universe(MULTIPDB_HOLE)
```